locales/sk/json.json

```diff
@@ -679,72 +679,72 @@
+    "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Títo ľudia boli pozvaní do vášho tímu a bol im zaslaný email s pozvánkou. Môžu sa pripojiť k tímu potvrdením emailovej pozvánky.",
     "This account does not have an active subscription.": "Tento účet nemá aktívne predplatné.",
     "This action is unauthorized.": "Táto akcia je neoprávnená.",
     "This copy of Nova is unlicensed.": "Táto kópia Nova je bez licencie.",
     "This coupon code can only be used by new customers.": "Tento kód kupónu môžu použiť iba noví zákazníci.",
     "This device": "Toto zariadenie",
-    "This file field is read-only.": "Toto pole súboru je len na čítanie.",
+    "This file field is read-only.": "Toto pole je určené len na čítanie.",
     "This invoice is no longer open.": "Táto faktúra už nie je otvorená.",
     "This is a secure area of the application. Please confirm your password before continuing.": "Toto je bezpečná oblasť aplikácie. Pred pokračovaním potvrďte svoje heslo.",
     "This password does not match our records.": "Toto heslo sa nezhoduje s našimi záznamami.",
     "This password reset link will expire in :count minutes.": "Platnosť resetovacieho odkazu je limitovaná v minútach (:count min)",
-    "This resource no longer exists": "Tento zdroj už neexistuje",
+    "This resource no longer exists": "Takáto položka neexistuje",
     "This subscription cannot be resumed. Please create a new subscription.": "Toto predplatné nie je možné obnoviť. Vytvorte nové predplatné.",
     "This subscription has expired and cannot be resumed. Please create a new subscription.": "Toto predplatné vypršalo a nie je možné ho obnoviť. Vytvorte nové predplatné.",
     "This user already belongs to the team.": "Tento používateľ už patrí do tímu.",
     "This user has already been invited to the team.": "Tento používateľ už bol pozvaný do tímu.",
     "Timor-Leste": "Východný Timor",
-    "to": "na",
-    "To": "Komu",
-    "To finish enabling two factor authentication, scan the following QR code using your phone's authenticator application or enter the setup key and provide the generated OTP code.": "Ak chcete dokončiť aktiváciu dvojfaktorovej autentifikácie, naskenujte nasledujúci QR kód pomocou autentifikačnej aplikácie vášho telefónu alebo zadajte kľúč nastavenia a zadajte vygenerovaný OTP kód.",
+    "to": "do",
+    "To": "Do",
+    "To finish enabling two factor authentication, scan the following QR code using your phone's authenticator application or enter the setup key and provide the generated OTP code.": "Ak chcete dokončiť aktiváciu dvojfaktorovej autentifikácie, naskenujte nasledujúci QR kód pomocou autentifikačnej aplikácie Vášho telefónu alebo zadajte kľúč nastavenia a zadajte vygenerovaný OTP kód.",
     "Today": "Dnes",
     "Toggle navigation": "Prepnúť navigáciu",
-    "Togo": "Ísť",
+    "Togo": "Togo",
```

Comment:
The value also needs to be added to the `_excludes.json` file, otherwise the status will show up as requiring translation.

Reply:
Added